.ko.yaml

```diff
@@ -1,7 +1,6 @@
 defaultBaseImage: gcr.io/distroless/static:nonroot
 baseImageOverrides:
-  # These base images run as root, which is needed for how they handle SSH credentials.
-  # They are produced from ./images/Dockerfile
+  # git-init resource supports to run as non-root user with UID 65532 that can clone repos without requiring root permissions.
```

Comment:
suggest `git-init uses a base image that supports running either as root or as user nonroot with UID 65532`

Reply:
Thanks for the suggestion @sbwsg !

Update by using this better one.